### ISSUE TITLE
Fix error with TestCliStatsNoStreamGetCpu

### ISSUE THREAD

**Issue by jfrazelle** [docker/docker#13785]
```
---------------------------------------------------------------------
FAIL: github.com/docker/docker/integration-cli/_test/_obj_test/docker_api_stats.go:12: DockerSuite.TestCliStatsNoStreamGetCpu

github.com/docker/docker/integration-cli/_test/_obj_test/docker_api_stats.go:18:
    ...open github.com/docker/docker/integration-cli/_test/_obj_test/docker_api_stats.go: no such file or directory
... value *errors.errorString = &errors.errorString{s:"error executing docker inspect: exit status 1\nError response from daemon: 404 page not found"} ("error executing docker inspect: exit status 1\nError response from daemon: 404 page not found")
```

**Comment by coolljt0725** [comment]
we should remove  '--cpu-quota=2000' of `out, _ := dockerCmd(c, "run", "-d", "--cpu-quota=2000", "busybox", "/bin/sh", "-c", "while true;do echo 'Hello';done")`  in this tests.

**Comment by coolljt0725** [comment]
I think change this line to `out, _ := dockerCmd(c, "run", "-d", "busybox", "/bin/sh", "-c", "while true; do echo 'hello'; sleep 0.1; done")` may fix this

**Comment by clnperez** [comment]
@coolljt0725 Do you want to work on this one? If not, I can grab it.

**Comment by coolljt0725** [comment]
@clnperez  no, just do it. thank you

**Comment by clnperez** [comment]
Thanks @coolljt0725 
#dibs

**Comment by clnperez** [comment]
@jfrazelle, which version of Docker were you running against? I just ran this with the latest upstream and it didn't fail for me.

**Comment by jfrazelle** [comment]
That's master it's still failing on test server

**Comment by clnperez** [comment]
So, I didn't realize this one had a history (PR #13645, for example). It looks like @coolljt0725 had seen those since he commented there. 

@jfrazelle , I'm not familiar with the test server exactly (except that tests are run when a PR is submitted). Is there some way to verify against that server other than submitting a PR? If there's any other info you think would be helpful, I'd appreciate it!

**Comment by clnperez** [comment]
tl;dl: Some more post mortem info would be helpful.Looking at this some more, it looks like maybe the container never started, or crashed. Definitely something went wrong outside the calculation. But since I can't get it to fail locally, it's hard to know. Maybe putting something in to re-try the run command if it doesn't exist. But I'm wondering why this test in particular is so flaky. It seems like other tests would run into start/crash issues as well.

**Comment by cpuguy83** [comment]
FYI, fixing this along with some other stats --no-stream fixes.

**Comment by clnperez** [comment]
Ah, cool. Can you take the dibs tag then? Or does it matter, as long as it's clear that someone is working on it?

**Comment by cpuguy83** [comment]
doesn't matter, it's just a label.
Fix is in #13911
